### ISSUE TITLE
Wrong icon on dashboard widget form approved submissions

### ISSUE THREAD

**Issue by ed-kffein** [engram-design/Workflow#136]
When listing the approved submissions with de dashboard widget, the status icon appears as a blank icon instead of a green icon. 

**Steps to reproduce**
1. Create a new widget "Workflow Submissions"
2. Choose Approved Submissions

**Additional info**
- Plugin version: 1.5.9
- Craft version: 3.6.17
- Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/workflow` requirement in `composer.json` to:

```
"require": {
  "verbb/workflow": "dev-craft-3 as 1.6.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.6.3](https://github.com/verbb/workflow/releases/tag/1.6.3)
